src/utils/CsvImporter.js

```diff
@@ -32,7 +32,7 @@ export function importCSV(file, restrictedValues, callback) {
   const field_defs  = {
     'id': {
       unique: true,
-      aliases: [ 'server_id', 'id' ],
+      aliases: [ 'server-id', 'id' ],
```

Comment:
should this include both server_id and server-id ?  @GarySmith ?

Reply:
aliases include the normalized key which has the - , it is consistent with the patterns for all other keys.

Reply:
thanks for clarifying!